### ISSUE TITLE
color disappear and window become small when switching between HSL and HSLuv

### ISSUE THREAD

**Issue by ghifarit53** [uga-rosa/ccc.nvim#9]
I got another issue. When I set the inputs to be HSL and HSLuv and also set `default_color`, switching between mode will make the color disappear and ccc window become small, making the slider disappear, and when its reappear, it doesn't work anymore

## Expected behavior
Color should be preserved and slider didn't disappear and still working

## Actual behavior
Color disappear, window become smaller, slider doesn't work and is gone

## Configuration
```lua
-- color picker
use {
  "uga-rosa/ccc.nvim",
  cmd = "CccPick",
  config = function()
    require("ccc").setup {
      default_color = "#40bfbf",
      inputs = {
        require("ccc.input.hsl"),
        require("ccc.input.hsluv"),
      },
    },
  end
}
```

## Steps to reproduce
1. Open ccc
2. Switch between modes several times and the issue will appear

## Attachments
![ccc-hsl-hsluv-issue](https://user-images.githubusercontent.com/59758342/190841055-31df8c04-8972-451b-bd4b-4220cd5d1917.gif)

**Comment by uga-rosa** [comment]
Fixed in 7650282f9e755b6219f6c58399102d9459050079.
I brought the official implementation for hsluv, but they represent RGB over there as 0 to 1 instead of 0 to 255. That correction was missing.

**Comment by ghifarit53** [comment]
That was fast, thanks!
